modules/ownKeys.js

```diff
@@ -0,0 +1,16 @@
+import objectKeys from 'object-keys'
+
+const ownKeys = (object) => {
+  if (Reflect && Reflect.ownKeys)
+    return Reflect.ownKeys(object)
+
+  const keys = objectKeys(object)
+
+  if (Object.getOwnPropertySymbols)
+    return keys.concat(Object.getOwnPropertySymbols(object))
```

Comment:
this will also include non-enumerable Symbols, whereas the other approaches exclude non-enumerable items - i think you want to filter the symbols array and exclude non-enumerables